lwgenerate/azure/azure.go

```diff
@@ -462,17 +456,6 @@ func createActivityLog(args *GenerateAzureTfConfigurationArgs) ([]*hclwrite.Bloc
 			attributes["all_subscriptions"] = args.AllSubscriptions
 		}
 
-		// Set storage account name, if set
-		if args.StorageAccountName != "" {
-			attributes["storage_account_name"] = args.StorageAccountName
```

Comment:
These inputs are available for Activity Log integrations.

Look at https://registry.terraform.io/modules/lacework/activity-log/azure/latest?tab=inputs

Reply:
Oh, gotcha, this is Activity log. Just updated.